### PR TITLE
[stable10] Release 10.1.1

### PR DESCRIPTION
**Pull request by PVince81**
- [x] solve conflicts (make sure we don't merge back the fix that was designed for 10.1.0)

### CODE REVIEW

**Comment by PVince81** [comment]
conflicts solved.

somehow had to remove the OccContext additions that don't seem to exist on stable10

File: .drone.yml [hunk 1]
```diff
@@ -200,7 +200,7 @@ pipeline:
       - OBJECTSTORE=${OBJECTSTORE}
     commands:
       - cd /drone/src/apps
-      - git clone https://github.com/owncloud/files_primary_s3.git
+      - git clone -b v1.0.3 https://github.com/owncloud/files_primary_s3.git
```
Comment (phil-davis) [inline .drone.yml]: We do not want to pin it like this in stable10

File: .drone.yml [hunk 2]
```diff
@@ -264,7 +264,7 @@ pipeline:
     image: owncloudci/php:${PHP_VERSION}
     pull: true
     commands:
-      - git clone https://github.com/owncloud/notifications.git apps/notifications
+      - git clone -b v10.1.1 https://github.com/owncloud/notifications.git apps/notifications
```
Comment (phil-davis) [inline .drone.yml]: We do not want to pin it like this in stable10

**Review by phil-davis — COMMENTED** [review]
IMO we do not want the pinning of app versions to get merged back to stable10. stable10 is for 10.2+ that works with whatever are "current" app versions.

Maybe instead of having this long list of commits in the branch/PR that do stuff and change parts of it back, we can just make a new ordinary branch and PR that cherry-picks just the drone "branches" line, and the changelog

**Comment by PVince81** [comment]
right... now considering that most changes here are not to be merged back, I guess I'll just cherry-pick the changelog in another PR

**Comment by PVince81** [comment]
please don't delete the branch

changelog only PR here: https://github.com/owncloud/core/pull/34908

**Pull request closed without merging**